scripts/build-json/resolve-status.js

```diff
@@ -0,0 +1,12 @@
+function packageStatus(specifications, bcd) {
+  const bcdHasStatus = !!bcd.__compat && !!bcd.__compat.status;
```

Comment:
`!!` is a bit cryptic. I'd prefer something more explicit here (`bcd.__compat !== undefined` or whatever we're getting at).

Also, the BCD schema requires `__compat` to have a `status` object. Seems like we should be able to trust BCD to not break Stumptown.

Reply:
Thanks @ddbeck ! I'm glad BCD mandates `status`, and have correspondingly tried to simplify this bit of the code.

Reply:
Of course this really wants optional chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) but that's not in Node yet apparently.